src/fmu/tools/rms/import_localmodules.py

```diff
@@ -109,3 +99,69 @@ def import_localmodule(project, module_root_name):
         xmod = importlib.import_module(module_root_name)
         sys.path.pop(0)  # avoid accumulation in sys.path by removing tmppath from stack
         return xmod
+
+
+@no_type_check
+def _localmodule_outside_rms(module_root_name, path):
+    """Import a library module in RMS which exists outside the RMS project."""
+
+    if not _detect_module_or_package(path, module_root_name):
+        raise ValueError(f"Cannot detect module {module_root_name}. Check spelling etc")
+
+    # Now empty sys.path for this module, allowing a refresh when library is modified:
+    sysm = sys.modules.copy()
+    for key, val in sysm.items():
+        if module_root_name in str(val):
```

Comment:
could this be checked against the key instead? it may be safer.

Reply:
Did also extend documentation